index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>My Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header class="navbar">
+      <a href="#">Dan Logo</a>
+      <span> <img src="images/burger.png" alt="toggle icon" /> </span>
```

Comment:
Nice work @Danelif ✔️

- [ ] [Optional] please try to surround your hamburger icon with an`<a>` tag. This is because this icon is also clickable